app/utilities/schema.py

```diff
@@ -180,4 +189,10 @@ def load_schema_from_url(survey_url, language_code):
 
 
 def get_schema_file_path(schema_name, language_code):
-    return SCHEMA_PATH_MAP.get(language_code, {}).get(schema_name)
+    return get_schema_path_map().get(language_code, {}).get(schema_name)
+
+
+def load_questionnaire_schemas_into_cache():
+    for language_code, schemas in get_schema_path_map(dirs=(SCHEMA_DIR,)).items():
+        for schema in schemas:
+            load_schema_from_name(schema, language_code)
```

Comment:
```suggestion
	[[load_schema_from_name(schema, language_code) for schema in schemas]
	    for language_code, schema in get_schema_path_map(dirs=(SCHEMA_DIR,)).items()]
```

Reply:
I didn't make it a comprehension as it wasn't returning and in this instance thought it was more readable.